### PR TITLE
New questions for Agri

### PR DESCRIPTION
**Pull request by loneil**
# Description
https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-1325
Rework questions around "Provide safe lodging and accommodation: General Worker" in Step2

## Types of changes

New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/frontend/src/components/agriseafoodopscreening/Step2.vue [hunk 1]
```diff
@@ -53,6 +53,13 @@
           :readonly="reviewMode"
           label="I have a plan to communicate physical distancing requirements to my employees (e.g. posting physical distancing signs in the accommodation area)."
         ></v-checkbox>
+
+        <v-checkbox
+          v-model="sharedSleepingProvidedAccomodations"
+          data-test="cb-form-sharedSleepingProvidedAccomodations"
```
Comment (parc-jason) [inline app/frontend/src/components/agriseafoodopscreening/Step2.vue:59]: variables/fields should also have 2 m - sharedSleepingProvidedAccommodations

File: app/src/db/migrations/20200616175436_014-agriseafoodopscreening-change-sleeping-fields.js [hunk 2]
```diff
@@ -5,20 +5,23 @@ exports.up = function (knex) {
...
     .then(() => knex.schema.alterTable(`${AGRI_PREFIX}_submission_attestation`, table => {
       table.boolean('sharedSleepingCommunication').notNullable().defaultTo(false);
+      table.boolean('sharedSleepingProvidedAccomodations').notNullable().defaultTo(false);
 
       table.dropColumn('sharedSleepingPerRoom');
       table.dropColumn('sleepingAreaType');
       table.dropColumn('commonAreaDistancing');
     }))
     .then(() => knex.schema.alterTable(`${AGRI_PREFIX}_submission_attestation`, table => {
-      table.boolean('sharedSleepingCommunication').notNullable().alter();
```
Comment (parc-jason) [inline app/src/db/migrations/20200616175436_014-agriseafoodopscreening-change-sleeping-fields.js]: shouldn't this still be in the script?  we need to run it for both new columns.

File: app/src/forms/attestations/agriseafoodopscreening/models/form.js [hunk 3]
```diff
@@ -127,8 +127,9 @@ class Attestation extends AttestationModels.AttestationTransportation {
     delete schemaProperties.sharedSleepingPerRoom;
     delete schemaProperties.sleepingAreaType;
     delete schemaProperties.commonAreaDistancing;
-    // and add the new one.
+    // and add the new ones.
```
Comment (parc-jason) [inline app/src/forms/attestations/agriseafoodopscreening/models/form.js:130]: ha - guess we will see how long this list of new ones gets...

File: app/src/db/migrations/20200616175436_014-agriseafoodopscreening-change-sleeping-fields.js (hunk 2 above)
Comment (loneil) [reply app/src/db/migrations/20200616175436_014-agriseafoodopscreening-change-sleeping-fields.js]: Just fixed this

File: app/frontend/src/components/agriseafoodopscreening/Step2.vue (hunk 1 above)
Comment (loneil) [reply app/frontend/src/components/agriseafoodopscreening/Step2.vue]: Fixed

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-vmvfjv-tools.pathfinder.gov.bc.ca/dashboard?id=common-forms-toolkit-pr-117

**Pull request merged**